openrtb_ext/bidders.go

```diff
@@ -36,6 +36,7 @@ const (
 	BidderReservedGeneral BidderName = "general" // Reserved for non-bidder specific messages when using a map keyed on the bidder name.
 	BidderReservedPrebid  BidderName = "prebid"  // Reserved for Prebid Server configuration.
 	BidderReservedSKAdN   BidderName = "skadn"   // Reserved for Apple's SKAdNetwork OpenRTB extension.
+	BidderReservedBidder  BidderName = "bidder"  // Reserved for Apple's SKAdNetwork OpenRTB extension.
```

Comment:
Nitpick: I believe the comment was simply copy pasted from the entry above. Can we elaborate a bit more?

Reply:
Updated the comment description.